src/data/faq.json

```diff
@@ -213,6 +213,27 @@
+                            "See also the following entry on <a href='#vac_cert_invalid' target='_self' rel='noopener'>invalid vaccination certificates</a>."
+                        ]
+                    },
+                    {
+                        "title": "Why do I get the error message 'VC_ALREADY_REGISTERED', 'TC_ALREADY_REGISTERED' or 'RC_ALREADY_REGISTERED' when scanning a certificate?",
+                        "anchor": "hc_already_registered",
+                        "active": false,
+                        "textblock": [
+                            "There have been an increasing number of messages reporting receiving this error message and not being able to find the scanned certificate. The reason for this error is that the certificate is already in the Corona-Warn-App, but it was saved under a 'different person'.",
+                            "<div class='container'><div class='row'><div class='col-sm'><figure><img class='img-fluid' src='../../assets/screenshots/2-16/en/android/PersonOverviewFragment_one_person.png' alt='image'></figure></div><div class='col-sm'><figure><img class='img-fluid' src='../../assets/screenshots/2-16/en/android/PersonDetailsFragment_cwa_2.png' alt='image'></figure></div><div class='col-sm'><figure><img class='img-fluid' src='../../assets/screenshots/2-16/en/ios/iPhone 13-screenshot_recovery_certificate_valid_details_part2.png' alt='image'></div></div></div>",
+                            "<b>How can this be checked?</b>",
+                            "In the Corona-Warn-App, the certificates are displayed and grouped at different levels: <ul><li><b>'Certificates' tab</b>: Under the 'Certificates' tab, users can see all existing certificates grouped by person. Check if multiple people are visible by scrolling down in this view. Errors often occur when issuing certificates, for example, if a second name or a title is used irregularly or the date of birth is given differently. Any discrepancies will result in certificates being assigned to a 'new person'.</li><li><b>All certificates of one person</b>: By tapping on a person's card, you can see all the certificates that are available for that person in the app. Check under which person the certificate you scanned can be found.</li><li><b>Certificate details</b>: By selecting a certificate, you can view details about the respective certificate. Check the standardized name and date of birth of the certificate. These data must be the same for all certificates so that they are displayed correctly under all certificates of a person.</li></ul>",
+                            "If you continue to receive the error message and have been able to verify that the certificate is indeed not already in your app, please reach out to us by mail at<a href='corona-warn-app.opensource...sap...com' class='email'>corona-warn-app.opensource...sap...com</a>. If your certificate has been issued incorrectly, please contact the issuer of the certificate."

```

Comment:
Missing space after "please reach out to us by mail at"

Reply:
Fixed in latest commit, thanks!